ksql-metastore/src/main/java/io/confluent/ksql/metastore/StructuredDataSource.java

```diff
@@ -76,6 +76,10 @@ public KsqlTopicSerDe getKsqlTopicSerde() {
     return ksqlTopic.getKsqlTopicSerDe();
   }
 
+  public boolean isAvroSerialized() {
```

Comment:
more generic if this method was:

```suggestion
  public boolean isFormat(final DataSourceSerDe format) {
```